$NetBSD$

Gracefully fallback to curses(3) (no libcurses `.pc' file is present
and `-l$libcurses' is used and hence `-lcurses' (not `-llibcurses'!)).

--- configure.orig	2017-07-23 17:38:55.000000000 +0000
+++ configure
@@ -308,7 +308,7 @@ int main(int argc, char *argv[]) {
 }
 EOF
 
-	for libcurses in ncursesw ncurses libcurses; do
+	for libcurses in ncursesw ncurses curses; do
 		printf " checking for %s... " "$libcurses"
 
 		if test "$have_pkgconfig" = "yes" ; then
